evidence/pool.go

```diff
@@ -424,25 +263,30 @@ func (evpool *Pool) fastCheck(ev types.Evidence) bool {
 			evpool.logger.Error("Failed to load evidence", "err", err, "evidence", lcae)
 			return false
 		}
-		evInfo, err := bytesToInfo(evBytes)
+		var trustedPb tmproto.LightClientAttackEvidence
+		err = trustedPb.Unmarshal(evBytes)
 		if err != nil {
-			evpool.logger.Error("Failed to convert evidence from proto", "err", err, "evidence", lcae)
+			evpool.logger.Error("Failed to convert evidence from bytes", "err", err, "evidence", lcae)
```

Comment:
same. I think we should print `key` here and maybe first 20 bytes of `evBytes`

Reply:
perhaps the height might be more helpful? We already know it is `LightClientAttackEvidence`

Reply:
key includes height, no?

Reply:
yeah true. I guess people debugging might not know that so I will try make it clear that the key is the height and the hash